packages/gatsby/src/commands/build-html.js

```diff
@@ -1,69 +1,76 @@
 /* @flow */
 const webpack = require(`webpack`)
 const fs = require(`fs`)
-const debug = require(`debug`)(`gatsby:html`)
 
 const webpackConfig = require(`../utils/webpack.config`)
-const { store } = require(`../redux`)
 const { createErrorFromString } = require(`gatsby-cli/lib/reporter/errors`)
 const renderHTMLQueue = require(`../utils/html-renderer-queue`)
 const telemetry = require(`gatsby-telemetry`)
 
-module.exports = async (program: any, activity: any) => {
+const runWebpack = compilerConfig =>
+  new Promise((resolve, reject) => {
+    webpack(compilerConfig).run((e, stats) => {
```

Comment:
I like explicit variables like err or error over e
```suggestion
    webpack(compilerConfig).run((err, stats) => {
```

Reply:
Addressed in 0ef8272